packages/backend/src/lib/backends/base.sync.ts

```diff
@@ -16,6 +16,9 @@
 
 import { IndexedInsight } from '@iex/models/indexed/indexed-insight';
 import { getLogger } from '@iex/shared/logger';
+import Container from 'typedi';
+
+import { UserService } from '@iex/backend/services/user.service';
```

Comment:
Is this needed here?
Seems like it's not being used.

Reply:
Good catch, cleaned up unused imports